src/Traverse.js

```diff
@@ -21,6 +21,10 @@ export default class ElementSearchIndex {
         return this._tokenIndex.select(type);
     }
 
+    selectTokensByValue(value: string): Array<Token> {
```

Comment:
This method already fails in JSCS for some tokens and will also fail for CST now. Selecting tokens by value without specifying type should not be possible. For instance, value of comment-token `/*+*/` is `+`, value of `//+` is also `+`, and TemplateStrings consist of several tokens with arbitary values.